doc/faq/installing_faq.rst

```diff
@@ -23,9 +23,9 @@ complexities. Open up a UNIX shell or a DOS command prompt and cd into a
 directory containing a minimal example in a file. Something like
 :file:`simple_plot.py` for example::
 
-  from pylab import *
-  plot([1,2,3])
-  show()
+  import matplotlib.pyplot as plt
+  plt.plot([1,2,3])
```

Comment:
I prefer using `subplots` too, but I figured the intro was intended to be hyper-brief.